aspnetcore/migration/inc/overview.md

```diff
@@ -62,9 +62,12 @@ Once the ASP.NET Framework app is no longer needed and deleted:
 
 ## System.Web Adapters
 
-The `Microsoft.AspNetCore.SystemWebAdapters` is a collection of runtime helpers that facilitate using code written against `System.Web` while moving to ASP.NET Core.
+The `Microsoft.AspNetCore.SystemWebAdapters` namespace is a collection of runtime helpers that facilitate using code written against `System.Web` while moving to ASP.NET Core. There are a few packages that may be used to use features from these adapters:
 
-The heart of the library is support for `System.Web.HttpContext`. The adapters attempt to provide compatible behavior for what is found running on ASP.NET Framework to expedite moving to ASP.NET Core. There are a number of behaviors that ASP.NET Framework provided that incur a performance cost if enabled on ASP.NET Core; these behaviors must be opted into.
+- `Microsoft.AspNetCore.SystemWebAdapters`: This package is used in supporting libraries and provide the System.Web APIs you may have taken a dependency on, such as `HttpContext` and others. This package targets .NET Standard 2.0, .NET 4.5+, and .NET 6+.
+- `Microsoft.AspNetCore.SystemWebAdapters.Abstractions`: This package is a supporting package that provides abstractions for shared services that the application may want to configure. This is generally not expected to be referenced from libraries, but rather from the applications themselves.
```

Comment:
Nit-pick: Move this to the end of the list since it's less likely to be referenced directly. Also, it's use is clearer after understanding the Core and Framework packages.